fe/fe-core/src/main/java/org/apache/doris/task/CreateReplicaTask.java

```diff
@@ -165,6 +202,10 @@ public TCreateTabletReq toThrift() {
         tSchema.setSchemaHash(schemaHash);
         tSchema.setKeysType(keysType.toThrift());
         tSchema.setStorageType(storageType);
+        if (dataSortInfo != null) {
+            tSchema.setSortType(dataSortInfo.getSortType());
+            tSchema.setSortColNum(dataSortInfo.getColNum());
```

Comment:
Better use default value instead of optional setting?

Reply:
dataSortInfo maybe null, will cause toThrift()  failed